MainRepository/Program.cs

```diff
@@ -1,7 +1,20 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿while (true) {
+   Console.Write ("Enter the total number of votes by the member ");
+   string input = Console.ReadLine ().ToLower ().Replace (" ", "");
+   if (input.ToLower () == "e") break;
+   if (input == "") Console.WriteLine ("Enter the valid input");
+   else {
+      VotingContest (input, out int totalVotes);
+      Console.WriteLine ($"The total votes is {totalVotes}");
+   }
+}
+
+static void VotingContest (string s, out int votes) {
+   Dictionary<char, int> voteCount = new ();
+   foreach (char c in s) {
+      var result = voteCount.ContainsKey (c) ? voteCount[c]++ : voteCount[c] = 1;

```

Comment:
Why you are using this way instead of if-else?

Reply:
@sdinesh-90 : Which one would be better? Please advise.

Reply:
As there is assignment, for clarity, it is better to use if else loop